core/src/main/java/com/hcl/ecomm/core/servlets/LoginServlet.java

```diff
@@ -37,11 +37,15 @@ protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse r
 		LOG.info("loginService : " + loginService);
 		
 		
-		String token = loginService.getToken();		
```

Comment:
@Subrat-Dash-416  why this change required in junit?

Reply:
same as above